Worksheet B/FalloutHack.xml

```diff
@@ -0,0 +1 @@
+<mxfile version="5.7.1.2" editor="www.draw.io" userAgent="Mozilla/5.0 (Windows NT 10.0; WOW64; Trident/7.0; .NET4.0C; .NET4.0E; .NET CLR 2.0.50727; .NET CLR 3.0.30729; .NET CLR 3.5.30729; McAfee; NMTE; rv:11.0) like Gecko" type="device"><diagram>5Vpdc5s6EP01fvUgBBgeY2K3fbj3ZsZ3ps1Th4JiNMGWRyix3V/fxUgGCXCNE5ykfQiRVujrnF3tavEIh6vdJx5t0n9YQrKRbS05TUb4dmTbFvyBYBWJVBP8iOLHJWdPa/neyMbz+cwLw7J5Ey3JZ0KXqSibEfKCquUrTdRwvu1V8kUcZUR2GLuVXJv5gWVyTlQKIs7ZNtdEMVuvSSy0foKxTNCN/uLyiSbEEMHeF/SnWofsnezLuuthKdipFxy7lORplLBtbU48G+GQMybK0moXkqxANxd7tc0RnqZilUEFQbEcx4B+ZM8v648UgJysxW+GfI6yJzlkLiIu5I6qiWBzm6K42i0LRRk/ZGwbp/DmWBC+outIMA4reKBZFrKsKEMvHARhOIe5prng7JHUWizLCYPg2KI0AoMEJkgoLLn2dhjO3MM4qu2WcuCXsjW0rxkv+k4fmNbHslCAsZQv5E7QKcDsJmASw2fCBdnVRBK9T4StiOD7QgulFi2PoqJjWrMBLNHfVtqPFCNSuzxZlVMd60fCoCA56+Tvvw1Zg+SOMwBr1YPIJBLRR6cQD0uh30KhG2gUImxw6FjqKOvD4i3NN1lUrOoznCm2taArmkUcSv+nBJ5fGU9y+H/DyV9GsTMsxeo0r1FsB65GsRP4GsXIv4DgRXrg9cgjOFJ45o/wnLOC57soz7fQ+pfR6w1Mr9ek19fptV1Hp3fi96f3S0GqOJhqRaQVMs4Pccm8D6skpvkB5Y/NrD/w2dw0XOR7GrPYQfrZjM413BpR5+LQwgnGntfg5GQMF3RiRhIVF0sBhG1LokINqV7sicekFtyciysnWSTosz7BuwVJxeh9UfKaKOGrogSr5ftvUC5CBDw9VO+LqqrcEU5hQsKl8LqwtlwgzoHVb8LqDQ9r7extnK3FchU8JPvBtrNKMD0IoCFlnP4EHKNMod/GzDhAXeSQHRXfJJJF+b7qDrW3pbL7anOSShXb1rn0Lz2gkdUSPVv6Bcj2VbAsj2js1x3CHaOw/Gqycl1SWo/QnIk2ysRt3nzNscp9N8fCyFzTxHPHODBdxw3nUR2ITTFS3rJ4tUZLvyi4ToszKgetFP8I+Lm28C9rWAO4WvEa/hwNfNlSXqymLo6uLY6rQ4jbILzCaXFdW+6+AJ20Zec6ptzTUo8j9bHMXVffy2zknuTDGcnA9xm7eZ8xjASrwHinmt/GSJR7tMaW7iEDd9LhIysvrPtgt8sFX9UQ8YVhZ0t85LwDQ2wkH11TT15imX29o28c7agtFfpi7/iKan1e1FeLKwPLfY9a7XZ7dUOr31hfEQp05++bV4XzYzzkGllb7DsvVGCEbX3IoBGDvoYG32TZIY94E8ckL3JOX9YCQj5rsc8F6ZP930AD/y4OzR87z+R2J01eJ9GEmirsGj7XiO2dyQUfckiREj6bvz/nM5w3GZa/M77DOVi/8XntkeYQzqXYIo1PJyHeyFFUEZznqJCtdF1+ZwhnUNudAz6dqZs0Y6bSzIdKRCAj/LjERZnpZs82nUoPD2WsB7Uq5En3AdXqxwjl69VvPvDsFw==</diagram></mxfile>
```

Comment:
The arrow coming out of "display how similar the words are" should go back to "show words and ask for password", to make it clear that the player is prompted for a new guess at this stage.

Also there needs to be a check if the player has used their allowed number of guesses, and if so end the game.